PokeAlarm/Utils.py

```diff
@@ -148,6 +148,69 @@ def get_move_energy(move_id):
             get_move_energy.info[int(id_)] = j[id_]['energy']
     return get_move_energy.info.get(move_id, 'unkn')
 
+# Returns the base stats for a pokemon
+def get_base_stats(pokemon_id):
+    if not hasattr(get_base_stats, 'info'):
+        get_base_stats.info = {}
+        file_ = get_path('locales/base_stats.json')
+        with open(file_, 'r') as f:
+            j = json.loads(f.read())
+        for id_ in j:
+            get_base_stats.info[int(id_)] = j[id_]
+    return get_base_stats.info.get(pokemon_id, {})
+
+# Returns the base height for a pokemon
+def get_base_height(pokemon_id):
+    return get_base_stats(pokemon_id).get('height')
+
+# Returns the base weight for a pokemon
+def get_base_weight(pokemon_id):
+    return get_base_stats(pokemon_id).get('weight')
+
+# Returns the size ratio of a pokemon
+def size_ratio(pokemon_id, height, weight):
+    height_ratio = height / get_base_height(pokemon_id)
+    weight_ratio = weight / get_base_weight(pokemon_id)
+    return height_ratio + weight_ratio
+
+# All available pokemon sizes
+class Size:
+    XS = 'XS'
+    SMALL = 'Small'
+    NORMAL = 'normal'
+    LARGE = 'Large'
+    XL = 'XL'
+    available_sizes = [XS, SMALL, NORMAL, LARGE, XL]
+
+
+# Returns normalized version of the sizes
+def get_normalized_size(raw_size):
```

Comment:
Because both `get_normalized_size` and `get_pokemon_size` need that string, I did not want to have the same string repeated, risking a typo that my break stuff. Just programming best pratices. Python does not help a lot in that sense, but anyway I think it is good to have some kind of constant.

The class is to help and avoid using the `global` keyword inside the functions.

I could also move the functions into Size and make them class methods.